src/fprime_gds/common/parsers/seq_file_parser.py

```diff
@@ -85,25 +82,24 @@ def parseArg(arg):
                 ):
                     return arg[1:-1]
                 # If the string contains a "." assume that it is a float:
-                elif "." in arg:
+                if "." in arg:
                     return float(arg)
-                elif arg in ["True", "true", "TRUE"]:
+                if arg in ["True", "true", "TRUE"]:
                     return True
-                elif arg in ["False", "false", "FALSE"]:
+                if arg in ["False", "false", "FALSE"]:
                     return False
-                else:
+                try:
+                    # See if it translates to an integer:
+                    return int(arg, 0)
+                except ValueError:
                     try:
-                        # See if it translates to an integer:
-                        return int(arg, 0)
+                        # See if it translates to a float:
+                        return float(arg)
                     except ValueError:
-                        try:
-                            # See if it translates to a float:
-                            return float(arg)
-                        except ValueError:
-                            # Otherwise it is an enum type:
-                            return str(arg)
+                        # Otherwise it is an enum type:
+                        return str(arg)
 
-            return list(map(parseArg, args))
+            return [map(parseArg, args)]
```

Comment:
I think this line is wrong.  This creates a list of one item which is the object returned by the `map` call.

list(map(...)) instead iterates the result of the map call and makes it a `list` type.  It is equivalent to:

```
[item for item in map(...)]
```

Reply:
Done!